src/components/shared/button/Button.styled.ts

```diff
@@ -0,0 +1,32 @@
+import styled from "styled-components";
+
+export const Primary = styled.button`
+  align-items: center;
+  background-color: #35ac45;
+  border-radius: 4px;
+  border: ${({ theme }) => theme.borderGreen};
+  color: #fff;
+  cursor: pointer;
+  display: ${(props) => (props.isFormVisible ? "none" : "flex")};
+  font-family: ProximaNova;
+  font-size: 14px;
+  height: 36px;
+  justify-content: center;
+  min-width: 100px;
+  padding: 0 20px;
+
+  :first-of-type,
+  :last-of-type {
+    margin-right: 12px;
+  }
```

Comment:
It's usually error-prone and advised against to add margins to shared/generic components. Can you solve this within the parent perhaps?

Reply:
I tried but couldn't really figure out how, I'll try again